torch/_inductor/decomposition.py

```diff
@@ -634,19 +748,23 @@ def miopen_batch_norm(
 
 
 @functools.lru_cache(None)
-def fast_random_decomps():
+def fast_random_decomps():  # type: ignore[no-untyped-def]
```

Comment:
`-> Dict[OpOverload, Callable[[Any], Any]` ?

Reply:
So I've tried this and a few external call sites have started breaking. Let me try again.

Reply:
So the best I could do is `Dict[Any, Callable[..., Any]]` without ripples in other files. Left a TODO.